extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -404,18 +405,16 @@ export class AmpSlideScroll extends BaseSlides {
   customSnap_(currentScrollLeft, opt_forceDir) {
     this.snappingInProgress_ = true;
     const newIndex = this.getNextSlideIndex_(currentScrollLeft);
-    let toScrollLeft;
+    // Default behavior should be stays on current slide
```

Comment:
@aghassemi do you mind double checking that this seems about right? I modified this because there was a possibility of it not being set, which runs afoul of type-check. I think the default behavior should be to stay on the same slide but would appreciate a sanity check.